README.md

```diff
@@ -1,8 +1,3 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
-
 
 ## Getting Started
```

Comment:
- [ ] Please, kindly add the title and description of the project in your readme file:

![image](https://user-images.githubusercontent.com/57421823/154196225-f9b55be3-e616-44ee-bc74-66f9e2af2420.png)

Just add and edit the title and description shown on the template to reflect the information of your project 💪